src/ResponseAnswersComponent.coffee

```diff
@@ -68,7 +69,7 @@ module.exports = class ResponseAnswersComponent extends AsyncLoadComponent
           "#{location.latitude}\u00B0 #{location.longitude}\u00B0"
           if location.accuracy then "(+/-) #{location.accuracy} m"
 
-  renderAnswer: (q, answer) ->
+  renderAnswer: (q, answer, dataId) ->
```

Comment:
What does the dataId argument do? Can you doc it?

Reply:
That I had used to check if coming from roster. Bot needed now!